pkg/apis/crd/v1beta1/types.go

```diff
@@ -151,3 +151,261 @@ type ControllerCondition struct {
 	// Human readable message indicating details
 	Message string `json:"message,omitempty"`
 }
+
+type TraceflowPhase string
+
+const (
+	// Pending is not used anymore
+	Pending   TraceflowPhase = "Pending"
```

Comment:
@gran-vmv  I suppose we can remove this in v1beta1 directly?

Reply:
Yes.
Currently it equals to empty string:
`case "", crdv1alpha1.Pending:`

Reply:
Removed